openapi/kas-fleet-manager-private-admin.yaml

```diff
@@ -325,6 +373,15 @@ components:
           type: boolean
     SupportedKafkaSizeBytesValueItem:
       $ref: 'kas-fleet-manager.yaml#/components/schemas/SupportedKafkaSizeBytesValueItem'
+    KafkacertificateRevocationRequest:
+      type: object
+      properties:
+        revocation_reason:
+          type: integer
+          description: The certificate revocation reason. See https://www.rfc-editor.org/rfc/rfc5280#section-5.3.1 for the available reasons
```

Comment:
Is the idea to explicitly couple this field to the x509 certificate revocation RFC intentional? Will the dataplane side need to understand that RFC?

Reply:
> Will the dataplane side need to understand that RFC?

This is not for the data plane. 
It is an admin management API - different from the agent-clusters API (kas-fleet-manager-private.yaml openapi file).

> Is the idea to explicitly couple this field to the x509 certificate revocation RFC intentional?

Yes - it was intentional.